sky/services/media/ios/media_player_impl.mm

```diff
@@ -58,6 +60,22 @@ - (BOOL)seekTo:(NSTimeInterval)interval {
   return [_player playAtTime:_player.deviceCurrentTime + interval];
 }
 
+- (void)setVolume:(double)volume {
+  if (volume > 1.0) {
```

Comment:
Good question.

We should document somewhere, maybe for now just as a comment in the mojom, what the range is and what the implementation's responsibilities are when receiving out-of-range values.